src/Components/index.js

```diff
@@ -1,3 +1,7 @@
 export { LandingPage } from './LandingPage/LandingPage';
 export { Header } from './Header/Header';
 export { Footer } from './Footer/Footer';
+export { NotesPage } from './NotesPage/NotesPage';
+export { Sidebar } from './Sidebar/Sidebar';
+export { ArchivePage } from './Archive/ArchivePage';
+export { TrashPage } from './Trash/TrashPage';
```

Comment:
I appreciate  this way of importing component in one file and export it from here